vue/dynamicforms/src/components/form/inputs/df-ckeditor.vue

```diff
@@ -9,27 +9,27 @@
   </vuetify-input>
 </template>
 
-<script lang="ts">
+<script setup lang="ts">
 import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
 import CKEditor from '@ckeditor/ckeditor5-vue';
-import { defineComponent } from 'vue';
 
-import TranslationsMixin from '../../util/translations-mixin';
-
-import InputBase from './base';
+import { BaseEmits, BaseProps, basePropsDefault, useInputBase } from './base';
 import VuetifyInput from './input-vuetify.vue';
 
-export default defineComponent({
-  name: 'DFWidgetCKEditor',
-  components: { ckeditor: CKEditor.component, VuetifyInput },
-  mixins: [InputBase, TranslationsMixin],
-  data() {
-    return {
-      editor: ClassicEditor,
-      editorConfig: {}, // The configuration of the editor.
-    };
-  },
-});
+const ckeditor = CKEditor.component;
+
+interface Props extends BaseProps {}
```

Comment:
Tole vem, da je v TS res težko, ampak a se tegale boilerplate-a res ne da zmanjšat? Tudi v tistih komponentah, kjer se propsi in emiti ne raztegujejo ne?

Reply:
Do nedavnega je bil odgovor: Ne. Se ko sem nazadnje testiral, to ni bilo mogoce.
Ocitno pa so v zadnjih verzijah slednje popravili in lahko uporabljas importane interface za definiranje propov as is.
Tako da se to tudi da popraviti.

Za emite obstaja se en issue -> trenutno se v consoli izpise kar nekaj warningov, sem odprl ticket na vue.